### PR TITLE
My feature branch

### PR DESCRIPTION
**Pull request by Asiimwe-Mark**
Set up a "Hello Microverse" project

General requirements
Once you finish your project, double-check to make sure your repo is ready:

Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct ([GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)).
Make sure that you documented your work.
Requirements
Remember to document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
Create a new GitHub repo.
Create a new branch according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules.
Add a .gitignore file.
Add node_modules to gitignore file.
Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
Add 3 files (each one in a separate commit):
A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
A .css file that includes one class with styles for your header (make it in your favorite color).
A test.md file with the text "This file should be ignored by git".
Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
Open a pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```
Comment (AbayomiOlaoye) [inline index.html]: - [ ] Well done, @As1imwe-Mark :hugs: I see that you have styled a class in your `css` file but you haven't included the class in the `<h1>` in the `index.html` file. The color is not being applied at the moment. See this :point_down: 
image

- _Kindly apply the class on the heading tag like `<h1 class="header">Hello Microverse</h1>`_

File: README.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] First, your ReadMe looks really nice in this very first attempt! But you see, to make your project look professional and convincing, it is important to remove all templates from your code. Personalizing your project content is really important. Please remove this. :wink:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+> Write a message to encourage readers to support your project
+
+If you like this project like it and give me a follow.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] First, your ReadMe looks really nice in this very first attempt! But you see, to make your project look professional and convincing, it is important to remove all templates from your code. Personalizing your project content is really important. Kindly remove any comment/description that is intended as a guide for what you should include in the respective section :pray:

File: README.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] First, your ReadMe looks really nice in this very first attempt! But you see, to make your project look professional and convincing, it is important to remove all templates from your code. Personalizing your project content is really important. Kindly remove any comment/description that is intended as a guide for what you should include in the respective section :pray:

File: README.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+- GitHub: @As1imwe-Mark
+- Twitter:@markasiimwe
+- LinkedIn:AsiimweMark
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] First, your ReadMe looks really nice in this very first attempt! But you see, to make your project look professional and convincing, it is important to remove all templates from your code. Personalizing your project content is really important. Kindly remove any comment/description that is intended as a guide for what you should include in the respective section :pray:

File: README.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+### Usage
+
+To run the project, execute the following command:
+-Double click the index.html file.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] It is important to make your project look professional and convincing, it is important to remove all templates from your code. Personalizing your project content is really important. Kindly remove any comment/description that is intended as a guide for what you should include in the respective section :pray:

File: README.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+### Key Features <a name="key-features"></a>
+- **Displays Hello Microverse**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (AbayomiOlaoye) [inline README.md]: - [ ] It is important to make your project look professional and convincing, it is important to remove all templates from your code. Personalizing your project content is really important. Kindly remove any comment/description that is intended as a guide for what you should include in the respective section :pray:

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @As1imwe-Mark :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- `.gitignore` file contains `test.md` and `node_module` files 
- Good use of GitHub flow :heavy_check_mark: 
- Good implementation of the header title :heavy_check_mark: 
- Files are structured cleanly :heavy_check_mark: 

## Required Changes ♻️
- Great job, and you are welcome to Microverse :+1: Let's see how can improve your Pull Requestion creation. It's is very important that project repositories are documented appropriately and Pull requests are in such a way that it informs the reviewer or any other person what the change you are requesting is all about.
  - Kindly read [this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) again to really have a clear idea of the importance of PR creation. I believe this will improve your PR from what we have now :point_down: :point_down: 
image
- [ ] [Kindly check this .gif image](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/images/pull_request_desc.gif)

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] As stated by the previous reviewer, kindly delete this section of the README file. You can see its there as a guide, and should not be ideally added which makes your code professional 👍

File: README.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+> Write a message to encourage readers to support your project
+
+If you like this project like it and give me a follow.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] As stated by the last reviewer, kindly remove this comment/description that is intended as a guide for what you should include. Add them to your codebase does not make your work professional 👍

File: README.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+- GitHub: @As1imwe-Mark
+- Twitter:@markasiimwe
+- LinkedIn:AsiimweMark
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] The last reviewer pointed out that you remove this section which serves as a comment/ description to guide you. Kindly do so to make your work look clean and proffestuonal 👍

File: README.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+### Usage
+
+To run the project, execute the following command:
+-Double click the index.html file.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please kindly check y comment in line `113` and make same changes here 👍

File: README.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+### Key Features <a name="key-features"></a>
+- **Displays Hello Microverse**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] kindly check y comment in line `113` and make same changes here 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @As1imwe-Mark     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  

  
# Required Changes  🌵 


- [x] You made some impressive implementations which I must say is a job well done. However, there are a few changes I need you to make In line with the requirements and the last reviwers comments
 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

- Kindly add a `features` and `Future Features` section on your README. This is very important to make your repo professional 👍 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @As1imwe-Mark👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- No linter errors✔️
- Professional README documentation✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**